modules/standard/Random.chpl

```diff
@@ -661,9 +673,8 @@ module Random {
     class RandomStream {
       /*
         Specifies the type of value generated by the PCGRandomStream.
-        Currently, only numeric types are supported, but all numeric
-        types should be supported (int, uint, of all sizes,
-        real, imag, and complex types of all sizes, and bool types).
+        All numeric types are supported: int, uint, of all sizes,
+        real, imag, and complex types of all sizes, and bool types.
```

Comment:
Probably bool types of all sizes are supported too.  :)  So this could simply say "bool, int, uint, real, imag, and complex types of all sizes."  I don't know if bools are really numeric types...

(Man, if only we added support for random voids and strings, we could claim all the primitive types are supported... :)  ).